vllm/v1/kv_cache_interface.py

```diff
@@ -122,6 +126,59 @@ def merge(cls, specs: list[Self]) -> Self:
             raise ValueError(
                 "All sliding window layers in the same KV cache group "
                 "must have the same window size.")
+        if len(attention_chunk_size) == 0:
+            merged_spec.attention_chunk_size = None
+        elif len(attention_chunk_size) == 1:
+            merged_spec.attention_chunk_size = attention_chunk_size.pop()
+        else:
+            raise ValueError(
+                "All chunked local attention layers in the same KV cache group "
+                "must have the same chunk size.")
+        assert len(sliding_window) + len(attention_chunk_size) <= 1, (
+            "Model with both sliding window layers and chunked local attention "
+            "layers is not supported.")
+        return merged_spec
+
+
+@dataclass
+class ChunkedLocalAttentionSpec(AttentionSpec):
+    attention_chunk_size: int
+
+    @property
+    def type_id(self) -> str:
+        return (f"local_attention_{self.block_size}_{self.page_size_bytes}")
```

Comment:
I think we shouldn't put layers with different chunk size to the same kv cache group. So I suggest to include `attention_chunk_size` into the `type_id` just like `sliding_window` in `SlidingWindowSpec`. Then, you don't need to implement `ChunkedLocalAttentionSpec.merge` as `KVCacheSpec.merge` should be able to achieve the same functionality.